### DISCUSSION TITLE
Developer Profile badge

### DISCUSSION THREAD

**Discussion by bampakoa** [google/devlibrary#398] [Q&A] [answered]
Hello,

I have already added the **Dev Library Contributor** badge to my Google Developer profile and now I received an email to redeem the **Dev Library Author** badge. Is it the same badge? I tried to redeem it but I got a message that I already have it.

**Comment by garimame** [comment, marked as answer]
Hey @bampakoa, it is the same badge you can ignore the email since you already have the badge :)
